2022/index.html

```diff
@@ -0,0 +1,1278 @@
+                </div>
+            </div>
+        </div>
+
+        <div class="row mt-5 mt-sm-6 mt-md-7">
+            <div class="col-12">
+                <h3 class="h2 mb-4 text-center"><span class="underline">Still missing you</span></h2>
+                <p class="text-center"> Mark, David, Louise and Ludo. </p>
+            </div>
+        </div>
+
+        <hr style="border-top: 0.3em solid #fff; opacity:0.3;" class="mt-5 mt-sm-6 mb-4 mb-sm-5">
+
+        <div class="row mt-5 mt-sm-6 mt-md-7 mb-4">
+            <div class="col-12">
+                <h3 class="h2 mb-3"><span class="underline">About our trustees</span></h3>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-6">
+                <h3 class="h4">Incoming</h3>
+                <p> Three new trustees: Devin, Onyeka and Jen </p>
+            </div>
+            <div class="col-sm-6">
+                <h3 class="h4">Outgoing</h3>
+                <p> Thank you and goodbye to trustee Julia</p>
+            </div>
+        </div>
+
+        <hr style="border-top: 0.3em solid #fff; opacity:0.3;" class="mt-5 mt-sm-6 mb-4 mb-sm-5">
+
+        <div class="row mt-5 mt-sm-6 mt-md-7">
+            <div class="col-sm-6 d-flex flex-column justify-content-center">
+                <h3 class="h2 mb-4"><span class="underline">Volunteers</span></h2>
+                <p> Four new additions to the volunteer team, Helen, John, Luis and Alison </p>
+            </div>
+
+            <div class="col-sm-6" style="opacity:0.3">
+                {% include image.html src="/assets/img/volunteer-icon.jpg" webp="/assets/img/volunteer-icon.webp" loading="lazy" width="300" height="300" class="rounded-circle" alt="Hand with a heart" %}
+            </div>
+        </div>
+    </div>
+</section>
+
+{% include section-slope.html topleft="mysoc-blue" bottomright="mysoc-green" %}
+
+<section class="bg-mysoc-green py-5 py-sm-6 py-md-7" id="climate" aria-label="Climate">
+    <div class="container">
+        <h2 class="h1 mb-3">Through our Climate programme</h2>
+        <p class="section-intro mb-6">We are helping councils, community organisations, campaign groups and individual citizens to take faster, more informed and effective action to cut emissions at the local level.</p>
+
+        <h3 class="h2 mt-5 mt-md-6 mb-4">We've packed a lot into 2022</h2>
+        <div class="row mt-5 mt-sm-6 mt-md-7">
+            <div class="col-sm-5">
+                <h4 class="h3 mb-4"><span class="underline">We went off the beaten track and explored new ways of working</span></h4>
+            </div>
+            <div class="col-sm-7">
+                <p>Through six rapid <a href="https://www.mysociety.org/climate/climate-prototyping/" class="black-link">prototyping weeks</a>, we convened groups of experts to help us imagine how we could solve big problems like…</p>
+            </div>
+        </div>
+        <div class="flex-card is--three">
```

Comment:
FYI @lucascumsille this `is--three` component looks ok at wide window widths, but at `xs`, `sm`, and `md` it looks really not great.

Reply:
Any reason we couldn’t just use [Bootstrap’s existing Card Decks](https://getbootstrap.com/docs/4.5/components/card/#card-decks) for this?

Reply:
(remember you’ve used this `is--three` in quite a few places throughout the page)